libs/banglejs/jswrap_bangle.c

```diff
@@ -1990,7 +1997,13 @@ void jswrap_banglejs_setLCDPowerBacklight(bool isOn) {
 #endif // !EMULATED
 }
 
-
+#if 0
```

Comment:
Please can you not add `#if 0` code to PRs that go into Espruino?

Reply:
Ok.